appcheck/firebase-appcheck-safetynet/firebase-appcheck-safetynet.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
 }
 
 firebaseLibrary {
+    libraryGroup "appcheck"
```

Comment:
Safetynet is deprecated and will shouldn't be released again under normal circumstances. 

https://firebase.google.com/docs/app-check/android/safetynet-provider